cloud/azure/database/mariadb/mode/discovery.pm

```diff
@@ -0,0 +1,144 @@
+#
+# Copyright 2022 Centreon (http://www.centreon.com/)
+#
+# Centreon is a full-fledged industry-strength solution that meets
+# the needs in IT infrastructure and application monitoring for
+# service performance.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+package cloud::azure::database::mariadb::mode::discovery;
```

Comment:
Please use this template to create your discovery mode: https://github.com/centreon/centreon-plugins/blob/master/cloud/azure/compute/aks/mode/discovery.pm

Reply:
I have just copied this from the Azure MySQL checks, I dont know what you mean. Sorry, I am not really a Perl developer :).